### ISSUE TITLE
Feature Request: Make Column Widths Adjustable

### ISSUE THREAD

**Issue by matheger** [veusz/veusz#430]
The columns in the data explorer and object browser panels are not adjustable in width. If data sets or objects have very long names or are nested at deep levels, one either has to resize the entire pane or double-click to edit the data set name. It would be useful to have the columns be adjustable in size, and have a horizontal scrollbar to manage the overflow.

**Comment by jeremysanders** [comment]
For the dataset editor, you can drag the separator between the two panes to change their relative size.

**Comment by matheger** [comment]
Yes, that's true. However I think it would be very useful to have the columns themselves be adjustable in width. Is there any specific reason for the current design?

**Comment by jeremysanders** [comment]
Well it makes it easier for the columns to automatically fill the window/pane width. This sort of thing is a bit of a pain with Qt.
